doc/index.rst

```diff
@@ -54,14 +46,16 @@ is slightly modified:
 
     python setup.py install --user
```

Comment:
I would strongly discourage use of the `--user` install.  Basically it has bitten me hard many times, because it is very easy to install there and then forget about [1].  Then some time later you run a load review tool and it is picking up the test version instead of the Ska-installed flight version.  I forget if the ACIS check tool reports the import path, but this is something that could be easy to miss. 

 (I have other gripes with `--user`, e.g. it completely fails with compiled packages if you use more than one Python binary, e.g. CIAO and Ska Python).

Since these packages don't need any build step and they are not namespace packages, a simple and safe solution is using `PYTHONPATH` to point to the git repo root.  For this module it doesn't seem you even have to bother with `python setup.py develop`.

[1] - even as recently as a few weeks ago I was seduced by the ease of a `--user` install for a quick temporary test, then got distracted, and sure enough one of our production cron tasks failed the next day because it was importing the test version.